video_xblock/static/js/studio-edit-transcripts-autoload.js

```diff
@@ -241,25 +218,25 @@ function removeEnabledTranscriptBlock(enabledTranscript, initialDefaultTranscrip
     allEnabledTranscripts = getDefaultTranscriptsArray('enabled');
     isSuccessfulRemoval = $.inArray(langCode, allEnabledTranscripts) === -1; // Is not in array
     isStoredVideoPlatform = $.inArray(langCode, initialDefaultTranscriptsLangCodes) !== -1;  // Is in array
-    // Display message with results on removal
+    // Display message with results of removal
     if (isSuccessfulRemoval && isStoredVideoPlatform) {
         showStatus(
             successMessageRemoval,
             'success',
-            '.api-request.remove-default-transcript.' + langCode + '.status-success',
-            '.api-request.remove-default-transcript.' + langCode + '.status-error');
+            $('.api-request.remove-default-transcript.' + langCode + '.status-success'),
```

Comment:
Create variables with data you need. And call function `showStatus` once at the end of function to avoid repetition